pyproject.toml

```diff
@@ -120,3 +102,33 @@ requires = [
     "openvino~=2025.2.0.dev"
 ]
 build-backend = "py_build_cmake.build"
+
+[dependency-groups]
+benchmark = [
+    "pandas>=2.0.0,<=2.2.3",
+    "seaborn>=0.10.0,<=0.13.2",
+    "tqdm>=4.50.0,<=4.67.1",
+]
+dev = [
+    "bandit>=1.7.0,<=1.8.3",
+    "jinja2>=3.0.0,<=3.1.6",
+    "pandas>=2.0.0,<=2.2.3",
+    "pytest>=7.0.0,<=8.3.5",
+    "pytest-harvest>=1.6.0,<=1.10.5",
+    "ruff>=0.1.0,<=0.11.7",
+]
+fuzzing = [
+    "atheris>=2.0.0,<=2.3.0",
+]
+
+[tool.uv]
+package = false
+default-groups = ["dev"]
+
+[tool.uv.sources]
+openvino = { index = "openvino-nightly" }
+
+[[tool.uv.index]]
+name = "openvino-nightly"
+url = "https://storage.openvinotoolkit.org/simple/wheels/nightly"
```

Comment:
is it possible to add pre-release as well ?

Reply:
yes, we can define 2 source links here

Reply:
tested, we could not set to sources for one package without marker condition, the same story for poetry